packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1484,9 +1448,9 @@ mixin WidgetInspectorService {
 
   Map<String, Object> _nodeToJson(
     DiagnosticsNode node,
-    _SerializeConfig config,
+    DiagnosticsSerialisationDelegate delegate,
```

Comment:
I believe Serialization instead of Serialisation is the spelling used in the rest of package:flutter